assets/app/scripts/directives/logViewer.js

```diff
@@ -224,6 +228,35 @@ angular.module('openshiftConsole')
               goChromeless: logLinks.chromelessLink,
               restartLogs: streamLogs
             });
+
+            APIDiscovery
+              .getLoggingURL()
+              .then(function(url) {
+                // TODO: update lodash for _.get, would rather do this:
+                //var projectName = _.get($scope.context, 'project', 'metadata', 'name');
+                var projectName = $scope.context &&
+                                  $scope.context.project &&
+                                  $scope.context.project.metadata &&
+                                  $scope.context.project.metadata.name;
+
+                var hasComponents = projectName && $scope.name && url;
+                if(!hasComponents) {
+                  return;
+                }
+
+                angular.extend($scope, {
+                  canViewArchive: true,
+                  kibanaAuthUrl: $sce.trustAsResourceUrl(url+'/auth/token'),
```

Comment:
`kibanaAuthUrl: $sce.trustAsResourceUrl(URI(url+'/auth/token').normalizePathname().toString()),`
seems to do the trick!